swarm/activations.py

```diff
@@ -8,3 +8,7 @@
 class xTanH(torch.nn.Module):
     def forward(self, x: torch.Tensor):
         return x - torch.tanh(x)
+
+class TanH(torch.nn.Module):
```

Comment:
yeah true. I saw the `torch.nn.Tanh` thing when I started researching to follow up ben's comment that he hadn't seen the x-tanh thing anywhere (which is true I couldn't see it either).  So what do you suggest for things like this in an 'unnecessary but probably harmless' category?  I think after writing this and getting it working I discovered the nn.Tanh and tried it but encountered trivial resistance, and didn't bother fixing it.

Reply:
I'm surprised you had trouble, using `activations.Tanh` or `torch.nn.Tanh` should work fine. I don't like unecessary code - with time you'll forget why it was there, and then the reasoning becomes it was there for a reason and never leaves. I usually like to be careful from the PR time since adding code complexity is trivial, removing complexity is really hard.